pkgs/development/python-modules/kanalizer/default.nix

```diff
@@ -0,0 +1,70 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchurl,
+  fetchFromGitHub,
+  rustPlatform,
+  pytestCheckHook,
+}:
+
+buildPythonPackage rec {
+  pname = "kanalizer";
+  version = "0.1.1";
+
```

Comment:
```suggestion
  pyproject = false;  # Built with cargo / maturin

```

Reply:
`pyproject = true` worked for me, while `pyproject = false` didn't install the package correctly.

Reply:
then true is good 😅